### DISCUSSION TITLE
assign multiple names

### DISCUSSION THREAD

**Discussion by TWART016** [photoprism/photoprism#2618] [General]
Hi,

is it possible to assign multiple names / people to a photo?

**Comment by graciousgrey** [comment]
If multiple faces are detected you can assign a name to every face --> https://docs.photoprism.app/user-guide/organize/people/

If no face has been detected it is not possible yet to assign people to an image. 
Improvements are planned:
https://github.com/photoprism/photoprism/issues/1863
https://github.com/photoprism/photoprism/issues/1548

**Reply by TWART016** [reply]
How can I check if there are multiple faces detected on a picture? On a picture  are clearly 2 persons. I want to add the name of person1 and person2 to this picture.

**Reply by graciousgrey** [reply]
See https://docs.photoprism.app/user-guide/organize/people/#__tabbed_1_2 :)
